easy-rules-core/src/test/java/org/jeasy/rules/api/RulesTest.java

```diff
@@ -129,6 +129,45 @@ public void size() {
         assertThat(rules.size()).isEqualTo(0);
     }
 
+    @Test
+    public void registerAll() {
+        Set<Rule> rulesSet = new HashSet<>();
+        rulesSet.add(new BasicRule("ruleA"));
+        rulesSet.add(new BasicRule("ruleB"));
+
+        rules.registerAll(rulesSet);
```

Comment:
As mentioned in #288, I don't see the added value of writing `rules.registerAll(rulesSet);` instead of:

```java
rulesSet.forEach(rules::register);
```